src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+For crew, the goal is to deconvert revolutionary converts, protect Heads from being converted, and ensure Head Revs are discovered and/or do not make it onto the evacuation shuttle. 
+
+### Win Conditions
+
+Win conditions for a round with a Revolutionary antag are the following:
+
+- Revolutionary Major Victory: No *alive, unrestrained* Command *on the evac shuttle* are non-rev, >**50%** of Command are converted Revolutionaries.
+- Revolutionary Minor Victory:  No *alive, unrestrained* Command *on the evac shuttle* are non-rev, <**50%** of Command are converted Revolutionaries.
+- Rev-Crew Tie: Both Rev Minor & Crew Minor criteria are met, or neither Rev Minor & Crew Minor criteria are met.
+- Crew Minor Victory: No *alive, unrestrained* headrevs are *on the evac shuttle* when Evac reaches CentComm.
+- Crew Major Victory: *All* revolutionaries are *deconverted* or *dead* when Evac reaches CentComm.
+
+The round-end screen could look like the following:
+
+```
+Revolutionary Major Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution.  [Green text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary-Crew Tie! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Green text]
+X% of Command were converted to the Revolution. [Green text]
+X Loyal Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Major Victory! [Green text]
+All Revolutionaries were killed or deconverted! [Red text]
+```
+
+## Conversion
+
+Conversion is how the Revolutionary antag faction grows, and is a *requirement* to obtain Revolutionary Major Victory. To convert a crewmember they must be interacted with by a Revolutionary over time, and once a critera is met they can be converted to the Revolution. Crew are also able to de-convert Revolutionaries. Conversion is not restricted to Headrevs, though they have access to certain tools that benefit them in this regard.
+
+To represent the convertable state a crewmember is in and the faction they belong to, there is now a hidden nummerical stat called Loyalty Health Points (LHP). 
```

Comment:
As discussed, this is for all intents and purposes a mental health stat. The main struggle I ran into with my old equivalent was that it's a bit of a magic number for the crew, and hard to explain to people without making it really obvious it's a Revs round.

Reply:
As of now, I am opposed to tying it into any mental health system, due to how integral it is to the gamemode.